actions/add-review-url/src/index.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import AddComment from './AddReviewUrl';
+import * as core from "@actions/core";
+
+async function run() {
+    const  addComment = new AddComment();
+    try {
+        const host = core.getInput('host');
+        let headref = '';

```

Comment:
I assume your goal here is to have headref be typed as `string`, not `string | undefined`. Try this:

```typescript
let headref = process.env.GITHUB_HEAD_REF || '';
```

It's more concise than using an if statement and the toString() (if it's a truthy value, then it's known to be a string).

Reply:
Thanks. Updated it.